docs/kyma/04-02-local-installation.md

```diff
@@ -128,81 +116,18 @@ To learn how to test Kyma, see [this](#details-testing-kyma) document.
 Use the `minikube.sh` script to restart the Minikube cluster without reinstalling Kyma. Follow these steps to stop and restart your cluster:
```

Comment:
I don't think you're using the `minikube.sh` script anymore, are you? :)

Reply:
Since the script is not used anymore, can we remove this line?